spec/parameters/id.yml

```diff
@@ -0,0 +1,5 @@
+name: id
+in: query
+description: Identifier up to which elements are returned.
```

Comment:
Depending on the ordering parameter, elements with id smaller than given id will be returned, or elements with id greater than given id will be returned. Being the first option the default option.

Maybe it affects the "up to" definition for the id.

https://github.com/nemtech/catapult-rest/blob/master/rest/src/db/CatapultDb.js#L152-L159